src/mscorlib/shared/System/IO/Path.Windows.cs

```diff
@@ -142,13 +149,21 @@ public static string GetPathRoot(string path)
             if (PathInternal.IsEffectivelyEmpty(path))
                 throw new ArgumentException(SR.Arg_PathEmpty, nameof(path));
 
+            return new string(GetPathRoot(path.AsReadOnlySpan()));
```

Comment:
ditto

Reply:
Wont it will then call the GetPathRoot(string)

Reply:
you're right, I misread.